### ISSUE TITLE
Enhancement - Save bars on change of spec/form/stealth/etc

### ISSUE THREAD

**Issue by raysmith59** [brittyazel/Neuron#275]
I noticed the last time I built my druid bars that the only way I could get things to save correctly was to log out after setting up buttons on each form bar individually.  Ideally it'd be good to be able to set up all of the forms at once and have it work correctly.

Probably somewhat related, I'd like to be able to dynamically alter a macro and have it take immediate effect, right now I'm having to relog for that to happen.

**Comment by brittyazel** [comment]
That's odd, I'm not seeing that issue on my druid. Let me try again and see if I can replicate it

**Comment by brittyazel** [comment]
I can't replicate this. I'm able to change my bar abilities and have them be retained upon changing specs. Also, I can change my macros in real time and have them immediately work. Is this true on all your toons? Can you disable all other addons as a test and see if it's still an issue?

**Comment by raysmith59** [comment]
Frankly, this is a legacy thing from back after the last database update that I'd bee meaning to submit for a while, it was doing it back then but since my bars are now set up I haven't tried again recently.  I'll re-initiate if I notice it again going forward.
